skimage/graph/tests/test_rag.py

```diff
@@ -206,6 +206,7 @@ def test_ncut_stable_subgraph():
     assert new_labels.max() == 0
 
 
+@pytest.mark.thread_unsafe
 def test_reproducibility():
```

Comment:
Clarification: Why isn't this test thread-safe? Am I missing something?

Reply:
This was a residual when the Cython-induced segfaults were appearing, I'll remove it